resources/qml/ApplicationSwitcher/ApplicationSwitcher.qml

```diff
@@ -0,0 +1,182 @@
+// Copyright (c) 2021 Ultimaker B.V.
+// Cura is released under the terms of the LGPLv3 or higher.
+
+import QtQuick 2.10
+import QtQuick.Controls 2.3
+import QtQuick.Layouts 1.15
+
+import UM 1.4 as UM
+import Cura 1.1 as Cura
+
+Item
+{
+    id: applicationSwitcherWidget
+    width: applicationSwitcherButton.width
+    height: width
+
+    Button
+    {
+        id: applicationSwitcherButton
+        anchors.verticalCenter: parent.verticalCenter
+        anchors.horizontalCenter: parent.horizontalCenter
+
+        width: Math.round(0.5 * UM.Theme.getSize("main_window_header").height)
+        height: width
+
+        background: Item
+        {
+            anchors.fill: parent
```

Comment:
No need to set this; https://doc.qt.io/qt-5/qml-qtquick-controls2-control.html#background-prop